src/DebugServer.php

```
@@ -1717,4 +1965,347 @@ private function getStackTrace(): array
+                        $displayValue = is_string($value) ? $value : json_encode($value);
+                        $this->log("  {$name} = {$displayValue}");
+                    }
+                }
+            }
+
+            if (isset($debugState['trace']['file'])) {
+                $this->log("📈 Trace file: {$debugState['trace']['file']}");
+                $this->log('📊 Trace lines: ' . count($debugState['trace']['content']));
+            }
+        }
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'outputComprehensiveDebugState'. (Unused Code Rules)

(UnusedPrivateMethod)